runtimevar/blobvar/blobvar.go

```diff
@@ -34,6 +51,90 @@ import (
 	"gocloud.dev/runtimevar/driver"
 )
 
+func init() {
+	runtimevar.DefaultURLMux().RegisterVariable(Scheme, &lazyBucketOpener{})
+}
+
+// Scheme is the URL scheme blobvar registers its URLOpener under on runtimevar.DefaultMux.
+const Scheme = "blob"
+
+type lazyBucketOpener struct {
+	mu      sync.Mutex
+	buckets map[string]*blob.Bucket
```

Comment:
Related to my suggestion below to have `URLOpener` use a `*blob.URLMux`: I would expect this sort of cache to be on the `URLOpener`, not the global default.

Reply:
Done.